src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -1098,27 +1098,44 @@ int shim::xclGetDeviceInfo2(xclDeviceInfo2 *info)
  */
 int shim::resetDevice(xclResetKind kind)
 {
+    int ret = 0;
+
     // Only XCL_USER_RESET is supported on user pf.
     if (kind != XCL_USER_RESET)
         return -EINVAL;
 
     std::string err;
     int dev_offline = 1;
-    int ret = mDev->ioctl(mUserHandle, DRM_IOCTL_XOCL_HOT_RESET);
+    ret = mDev->ioctl(mUserHandle, DRM_IOCTL_XOCL_HOT_RESET);
```

Comment:
Why was declaration moved to 1101?  Declare and defined variables where they are first used.